dts/riscv/wch/ch32v0/ch32v003a4m.dtsi

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 Michael Hope
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <wch/ch32v0/ch32v003.dtsi>
+
+&gpioa {
+	gpio-reserved-ranges = <0 1>, <3 5>;
+};
```

Comment:
Looks like this node stays the same in all dtsis; it probably should be moved to the common dtsi

Reply:
Done.